test/runner/lib/cover.py

```diff
@@ -154,6 +154,18 @@ def command_coverage_report(args):
         if args.show_missing:
             options.append('--show-missing')
 
+        if args.include:
+            options.extend([
+                '--include',
+                args.include
```

Comment:
Style recommendation....

Since this option (and the following one) only take a single argument, it would be easier to read as:

```python
options.extend(['--include', args.include])
```

Keeping the option and the arg together is also more readable with multiple options (doesn't apply in this case):

```python
options.extend([
    '--include', args.include,
    '--some-other-arg', args.some_other_arg,
])
```